rest/data/commit_queue.go

```diff
@@ -388,3 +488,30 @@ func GetAdditionalPatches(patchId string) ([]string, error) {
 		Message:    errors.Errorf("patch '%s' not found in commit queue", patchId).Error(),
 	}
 }
+
+func requireSigned(ctx context.Context, userRepo UserRepoInfo, prNum int) error {
+	settings, err := evergreen.GetConfig()
```

Comment:
Not that important, but we typically try not loading the settings via `evergreen.GetConfig()` unless there's a specific reason, and instead use the `evergreen.GetEnvironment().Settings()` (which is already passed into `EnqueuePRToCommitQueue`). Can we do that here, or is it critical to load the current GitHub token without restarting the app servers?

Reply:
Not critical. Done